mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -24,33 +24,29 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
-import logging
-import os
-from PyQt5 import QtWidgets, QtCore
+from PyQt5 import QtWidgets, QtGui, QtCore
 from mslib.msui.qt5 import ui_multiple_flightpath_dockwidget as ui
-from mslib.utils.qt import get_open_filenames
-from mslib.utils.config import load_settings_qsettings, save_settings_qsettings
-from fs import open_fs
-import mslib.msui.flighttrack as ft
 from mslib.msui import msui
-import xml
+import threading
```

Comment:
have a look on https://www.pythonguis.com/tutorials/multithreading-pyqt-applications-qthreadpool/
 QThreadPool()

for  QtCore.QThread() we have a worker in https://github.com/Open-MSS/MSS/blob/develop/mslib/utils/qt.py#L370
https://www.youtube.com/watch?v=1FZO4VKANbM